app/src/main/java/org/mozilla/fenix/onboarding/SyncCFRPresenter.kt

```diff
@@ -0,0 +1,68 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.onboarding
+
+import android.content.Context
+import android.view.View
+import androidx.recyclerview.widget.RecyclerView
+import mozilla.components.service.glean.private.NoExtras
+import org.mozilla.fenix.GleanMetrics.Onboarding
+import org.mozilla.fenix.R
+import org.mozilla.fenix.compose.cfr.CFRPopup
+import org.mozilla.fenix.compose.cfr.CFRPopupProperties
+import org.mozilla.fenix.home.recentsyncedtabs.view.RecentSyncedTabViewHolder
+
+/**
+ * Delegate for handling sync onboarding CFR.
+ *
+ * @param context used for various Android interactions.
+ * @param recyclerView will serve as anchor for the sync CFR
+ */
+
+class SyncCFRPresenter(
+    private val context: Context,
+    private val recyclerView: RecyclerView,
+) {
+
+    private var syncCFR: CFRPopup? = null
+
+    /**
+     * Check if [view] is available to show sync CFR.
+     */
+    fun showSyncCFR() {
+        findSyncTabsView()?.let {
+            CFRPopup(
+                text = context.getString(R.string.sync_cfr_message),
+                anchor = it,
+                properties = CFRPopupProperties(
+                    indicatorDirection = CFRPopup.IndicatorDirection.DOWN
+                ),
+                onDismiss = {
+                    when (it) {
+                        true -> Onboarding.syncCfrExplicitDismissal.record(NoExtras())
+                        false -> Onboarding.syncCfrImplicitDismissal.record(NoExtras())
+                    }
+                }
+            ) {
+            }.apply {
```

Comment:
```suggestion
            ).apply {
```
I think you can avoid the `{}` for `action` parameter since it default to `{}`. It looks a bit odd with the empty `{}` body here so I would suggest removing it or adding it as a named parameter if it's not possible.